src/config/dynamic-theme-fixes.config

```diff
@@ -8043,6 +8043,18 @@ CSS
 
 ================================
 
+sephora.com
+
+INVERT
+img[alt="Sephora"]
+svg.css-9uy14h
```

Comment:
`-9uy14h`
and all the other random nonsense can break within weeks/months. I consider to have something else as an identifier that isn't random.

Reply:
I have submitted an updated pull request with descriptive names instead of the random nonsense.
Pull request #5403